tests/smoke/object_test.go

```diff
@@ -142,10 +147,54 @@ func getBucket(bucketname string, bucketList []model.ObjectBucket) (model.Object
 	return model.ObjectBucket{}, errors.New("Bucket not found")
 }
 
+func (suite *SmokeSuitek8s) createObjectStore() (string, error) {
+	suite.helper.GetObjectClient().ObjectCreate()
+	time.Sleep(time.Second * 2) //wait for rgw service to to started
+	if suite.k8sh.IsServiceUpInNameSpace("rook-ceph-rgw") {
+		_, err := suite.k8sh.GetService("rgw-external")
+		if err != nil {
+			suite.k8sh.KubectlWithStdin(getRGWExternalServiceDef(), []string{"create", "-f", "-"}...)
+			if !suite.k8sh.IsServiceUpInNameSpace("rgw-external") {
+				return "Couldn't create object store ", fmt.Errorf("Cannot expose rgw servie for external user")
+			}
+
+		}
+
+		return "OJECT STORE CREATED", nil
+	}
+	return "ERROR", fmt.Errorf("RGW service not started")
```

Comment:
this func should only return error, no need for string. Just write the message to the log here instead of returning a message